### DISCUSSION TITLE
missing template repository file

### DISCUSSION THREAD

**Discussion by leifbin** [labring/sealos#4517] [General]
│ Listening on port 3000 url: http://template-frontend-7c6fd86d74-qxm4c:3000/                                                                                                                                  │
│ jsonRes error------------                                                                                                                                                                                   │
│   missing template repository file                                                                                                                                                                          │
│ git operation timed out                                                                                                                                                                                     │
│ git operation timed out                                                                                                                                                                                     │
│ jsonRes error------------                                                                                                                                                                                   │
│   missing template repository file                                                                                                                                                                          │
│ git operation timed out

**Comment by leifbin** [comment]
Does anyone know why?

**Comment by bxy4543** [comment]
@xudaotutou Help answer
